lib/workers/pr/index.ts

```diff
@@ -143,7 +143,10 @@ export async function ensurePr(
     logger.debug(
       `Branch is configured for branch automerge, branch status) is: ${await getBranchStatus()}`
     );
-    if ((await getBranchStatus()) === BranchStatus.yellow) {
+    if (
+      !config.stabilityDays &&
```

Comment:
what value whill `stabilityDays` when reached? Will stabilityDays  always suppress  this check?

Reply:
I believe it will still be the same value. If the branch is still pending due to some *other* check then it will remain that way forever.